test/validation-controller-factory.ts

```diff
@@ -1,22 +1,28 @@
-import { Container, Optional } from 'aurelia-dependency-injection';
-import {
-  ValidationControllerFactory,
-  ValidationController,
-  Validator
-} from '../src/aurelia-validation';
-
-describe('ValidationControllerFactory', () => {
-  it('createForCurrentScope', () => {
-    const container = new Container();
-    const standardValidator = {};
-    container.registerInstance(Validator, standardValidator);
-    const childContainer = container.createChild();
-    const factory = childContainer.get(ValidationControllerFactory);
-    const controller = factory.createForCurrentScope();
-    expect(controller.validator).toBe(standardValidator);
-    expect(container.get(Optional.of(ValidationController))).toBe(null);
-    expect(childContainer.get(Optional.of(ValidationController))).toBe(controller);
-    const customValidator = {};
-    expect(factory.createForCurrentScope(customValidator).validator).toBe(customValidator);
-  });
-});
+import { Container, Optional } from 'aurelia-dependency-injection';
+import {
+  AureliaValidationConfiguration,
+  ValidationControllerFactory,
+  ValidationController,
+  Validator,
+  validateTrigger
+} from '../src/aurelia-validation';
+
+describe('ValidationControllerFactory', () => {
+  it('createForCurrentScope', () => {

```

Comment:
With the changes suggested above, let's have a lightweight test to ensure validation controller can still have the right validate trigger value, even when instantiated with only 2 parameters